### ISSUE TITLE
1.0.3 does not run with error 

### ISSUE THREAD

**Issue by potmo** [yuzutech/annotations-action#60]
When running this action I get this error:
```
Run Attest/annotations-action@v1.0.3
/Users/runner/work/_actions/Attest/annotations-action/v1.0.3/dist/index.js:113
    return navigator.userAgent;
    ^

ReferenceError: navigator is not defined
    at Object.getUserAgent (/Users/runner/work/_actions/Attest/annotations-action/v1.0.3/dist/index.js:113:5)
    at /Users/runner/work/_actions/Attest/annotations-action/v1.0.3/dist/index.js:480:[6](https://github.com/resolutiongames-restricted/vikingghost/pull/278/checks#step:7:7)1
    at createCommonjsModule (/Users/runner/work/_actions/Attest/annotations-action/v1.0.3/dist/index.js:60:35)
    at Object.<anonymous> (/Users/runner/work/_actions/Attest/annotations-action/v1.0.3/dist/index.js:121:16)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:102[7](https://github.com/resolutiongames-restricted/vikingghost/pull/278/checks#step:7:8):10)
    at Module.load (internal/modules/cjs/loader.js:[8](https://github.com/resolutiongames-restricted/vikingghost/pull/278/checks#step:7:9)63:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:[12](https://github.com/resolutiongames-restricted/vikingghost/pull/278/checks#step:7:13))
    at internal/main/run_main_module.js:[17](https://github.com/resolutiongames-restricted/vikingghost/pull/278/checks#step:7:18):47
```

**Comment by potmo** [comment]
Sorry I saw now that 1.4.0 was latest. For some reason marketplace said that 1.0.3 was

**Comment by Mogztter** [comment]
> Sorry I saw now that 0.4.0 was latest. For some reason marketplace said that 1.0.3 was

That's weird... I cannot see version 1.0.3 on https://github.com/marketplace/actions/annotate-action
Do you still see it? Can you take a screenshot?

**Comment by potmo** [comment]
Sorry again. I'm just confused. From the marketplace I downloaded `Attest/annotations-action` and not `yuzutech/annotations-action` so thats why nothing made any sense. They have the same name and when searching for `annotaions-action` of marketplace Attests version pops up first (https://github.com/marketplace?type=actions&query=annotations-action+). I have now started to use `yuzutech/annotations-action@v0.4.0` which was the one I intended to use all along.
